recipes/nanorange/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+        "20191001":
```

Comment:
```suggestion
  "20191001":
```

Reply:
Why?

Reply:
Usual indentation for YAML files is done with 2 spaces

Reply:
Ok, I was not aware of  YAML conventions